PillarChap/Pillar.pillar

```diff
@@ -263,7 +287,7 @@ invisible and links will be rendered as: *@chapterAndSections*.
 
 An empty line starts a new paragraph.
 
-An annotated paragraph starts a line with ==\@@== followed by either ==todo== or ==note==. For example,
+An annotated paragraph starts a line with ==\@@== followed by a key word as ==todo== or ==note==. For example,
```

Comment:
keyword such as...

Any keyword is accepted but this requires configuration in the exported documents (e.g., some CSS rules for HTML export, some environments for LaTeX exports).